charts/lws/README.md

```diff
@@ -65,24 +65,26 @@ for more information on installing LWS with metrics using our Helm chart.
+| `nameOverride`                             | nameOverride                                   | ``                                                |
+| `fullnameOverride`                         | fullnameOverride                               | ``                                                |
+| `enablePrometheus`                         | enable Prometheus                              | `false`                                           |
+| `enableCertManager`                        | enable CertManager                             | `false`                                           |
+| `imagePullSecrets`                         | Image pull secrets                             | `[]`                                              |
+| `image.manager.repository`                 | Repository for manager image                   | `us-central1-docker.pkg.dev/k8s-staging-images/lws` |
+| `image.manager.tag`                        | Tag for manager image                          | `main`                                            |
+| `image.manager.pullPolicy`                 | Pull policy for manager image                  | `IfNotPresent`                                    |
+| `podAnnotations`                           | Annotations for pods                           | `{}`                                              |
+| `podSecurityContext.runAsNonRoot`          | Run pod as non-root user                       | `true`                                            |
+| `securityContext.allowPrivilegeEscalation` | Allow privilege escalation in security context | `false`                                           |
+| `securityContext.capabilities.drop`        | Drop all capabilities in security context      | `["ALL"]`                                         |
+| `service.type`                             | Type of lws controller service                 | `ClusterIP`                                       |
+| `service.port`                             | Lws controller service port                    | `9443`                                            |
+| `resources.requests.cpu`                   | CPU request for resources                      | `1`                                               |
+| `resources.requests.memory`                | Memory request for resources                   | `1Gi`                                             |
+| `nodeSelector`                             | Node selector                                  | `{}`                                              |
+| `tolerations`                              | Tolerations                                    | `{}`                                              |
+| `affinity`                                 | Affinity                                       | `{}`                                              |
+| `gangScheduling.enable`                    | enable gang scheduling capabilities            | `false`                                           |
```

Comment:
If we agree with the field change above, I think we should polish here as well.

Reply:
OK I have modified it, please check